### PR TITLE
Add vgg16 benchmark configuration

### PR DESCRIPTION
**Pull request by kuke**
Resolve [#6605](https://github.com/PaddlePaddle/Paddle/issues/6605) 
Don't merge. The performance of ResNet and VGG16 based on fluid both seem weird in the test environment. Need to fix that first.

### CODE REVIEW

File: fluid/vgg16.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        num_samples = 0
+        accuracy.reset(exe)
+        for batch_id, data in enumerate(train_reader()):
+            img_data = np.array(map(lambda x: x[0].reshape(data_shape),
+                                    data)).astype("float32")
+            y_data = np.array(map(lambda x: x[1], data)).astype("int64")
+            batch_size = 1
+            for i in y_data.shape:
+                batch_size = batch_size * i
+            y_data = y_data.reshape([batch_size, 1])
```
Comment (chengduoZH) [inline fluid/vgg16.py]: ->`y_data = y_data.reshape([-1, 1])`

File: fluid/vgg16.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        for batch_id, data in enumerate(test_reader()):
+            img_data = np.array(map(lambda x: x[0].reshape(data_shape),
+                                    data)).astype("float32")
+            y_data = np.array(map(lambda x: x[1], data)).astype("int64")
+            y_data = y_data.reshape([len(y_data), 1])
+
+            exe.run(inference_program,
+                    feed={"pixel": img_data,
+                          "label": y_data},
+                    fetch_list=[avg_cost] + test_accuracy.metrics)
```
Comment (chengduoZH) [inline fluid/vgg16.py]: 这里可以仿照一下fluid/mnist.py，fluid/resnet.py的方式

Comment (kuke) [reply fluid/vgg16.py]: 因为用了dropout，此处必须用inference program

File: fluid/vgg16.py (hunk 1 above)
Comment (kuke) [reply fluid/vgg16.py]: Done

**Pull request merged**